.travis.yml

```diff
@@ -11,6 +11,13 @@ env:
     global:
         - NUM_THREADS=4
 
+cache:
+    timeout: 10080 # 1 week
```

Comment:
according to https://docs.travis-ci.com/user/caching/ the `timeout` is expressed in seconds and `10080s=2.8h`. Am I not looking at the right doc?